### DISCUSSION TITLE
How to check whether my engraver supported?

### DISCUSSION THREAD

**Discussion by plushechnik** [arkypita/LaserGRBL#1250] [Q&A]
Sorry completely newbie and got lost and confused.
I've got that machine https://www.aliexpress.com/item/32953212422.html and looking for better software than original.
![image](https://user-images.githubusercontent.com/3198720/103648243-38502600-4f65-11eb-9ef3-d267ef2b7280.png)
It seems to be CH341 driver and I see some references about here but can't pull all together.

**Comment by arkypita** [comment]
https://github.com/arkypita/LaserGRBL/discussions/1214

**Reply by plushechnik** [reply]
Many thanks, looking into ...
